bundle/edu.gemini.pot/src/main/scala/edu/gemini/pot/ModelConverters.scala

```diff
@@ -0,0 +1,141 @@
+      case edu.gemini.shared.skyobject.Magnitude.Band.L  => MagnitudeBand.L
+      case edu.gemini.shared.skyobject.Magnitude.Band.M  => MagnitudeBand.M
+      case edu.gemini.shared.skyobject.Magnitude.Band.N  => MagnitudeBand.N
+      case edu.gemini.shared.skyobject.Magnitude.Band.Q  => MagnitudeBand.Q
+      case edu.gemini.shared.skyobject.Magnitude.Band.AP => MagnitudeBand.AP
+    }
+  }
+
+  implicit class NewMagnitudeBand2Old(val band: MagnitudeBand) {
+    def toOldModel: skyobject.Magnitude.Band = band match {
+      case MagnitudeBand._u => edu.gemini.shared.skyobject.Magnitude.Band.u
+      case MagnitudeBand._g => edu.gemini.shared.skyobject.Magnitude.Band.g
+      case MagnitudeBand._r => edu.gemini.shared.skyobject.Magnitude.Band.r
+      case MagnitudeBand._i => edu.gemini.shared.skyobject.Magnitude.Band.i
+      case MagnitudeBand._z => edu.gemini.shared.skyobject.Magnitude.Band.z
+
+      case MagnitudeBand.U  => edu.gemini.shared.skyobject.Magnitude.Band.U
+      case MagnitudeBand.B  => edu.gemini.shared.skyobject.Magnitude.Band.B
+      case MagnitudeBand.G  => edu.gemini.shared.skyobject.Magnitude.Band.g
+      case MagnitudeBand.V  => edu.gemini.shared.skyobject.Magnitude.Band.V
+      case MagnitudeBand.UC => edu.gemini.shared.skyobject.Magnitude.Band.UC
+      case MagnitudeBand.R  => edu.gemini.shared.skyobject.Magnitude.Band.R
+      case MagnitudeBand.I  => edu.gemini.shared.skyobject.Magnitude.Band.I
+      case MagnitudeBand.Z  => edu.gemini.shared.skyobject.Magnitude.Band.z
+      case MagnitudeBand.Y  => edu.gemini.shared.skyobject.Magnitude.Band.Y
+      case MagnitudeBand.J  => edu.gemini.shared.skyobject.Magnitude.Band.J
+      case MagnitudeBand.H  => edu.gemini.shared.skyobject.Magnitude.Band.H
+      case MagnitudeBand.K  => edu.gemini.shared.skyobject.Magnitude.Band.K
+      case MagnitudeBand.L  => edu.gemini.shared.skyobject.Magnitude.Band.L
+      case MagnitudeBand.M  => edu.gemini.shared.skyobject.Magnitude.Band.M
+      case MagnitudeBand.N  => edu.gemini.shared.skyobject.Magnitude.Band.N
+      case MagnitudeBand.Q  => edu.gemini.shared.skyobject.Magnitude.Band.Q
+
+      case MagnitudeBand.AP => edu.gemini.shared.skyobject.Magnitude.Band.AP
+    }
+  }
+
+  implicit class OldMagnitude2New(val m: skyobject.Magnitude) extends AnyVal {
+    def toNewModel: Magnitude = new Magnitude(m.getBrightness, m.getBand.toNewModel)
+  }
+
+  implicit class NewMagnitude2Old(val m: Magnitude) extends AnyVal {
+    def toOldModel: skyobject.Magnitude = new skyobject.Magnitude(m.band.toOldModel, m.value)
+  }
+
+  implicit class HmsDegCoords2Coordinates(val c: skyobject.coords.HmsDegCoordinates) extends AnyVal {
+    def toNewModel: Coordinates = Coordinates(RightAscension.fromAngle(c.getRa.toNewModel), Declination.fromAngle(c.getDec.toNewModel).getOrElse(Declination.zero))
+  }
+
+  implicit class SiderealTarget2SkyObject(val st:SiderealTarget) extends AnyVal {
```

Comment:
On the 3 target converts we are missing translating the proper motion. There is code on AGS to do that but it must be ported to this class once it is merged into the main trunk